omod/src/main/java/org/openmrs/module/appointments/web/controller/AppointmentController.java

```diff
@@ -77,6 +79,10 @@ public ResponseEntity<Object> saveAppointment(@Valid @RequestBody AppointmentReq
              * appointmentMapper should be fixed and validateAndSave() signature should be changed.
              */
             Appointment appointment = appointmentsService.validateAndSave(() -> appointmentMapper.fromRequest(appointmentRequest));
+            AdministrationService administrationService = Context.getService(AdministrationService.class);
+            boolean bookSMS = Boolean.valueOf(administrationService.getGlobalPropertyObject("sms.enableAppointmentBookingSMSAlert").getPropertyValue());
```

Comment:
where is this property defined? is this configured through config.xml and provided a default value false?

Reply:
This is done through clinic.config should it be defined here and initialised in clinic-config ?